app/models/atmosphere/fund.rb

```diff
@@ -48,7 +48,10 @@ class Fund < ActiveRecord::Base
               presence: true
 
     validates :balance,
-              numericality: true
+              numericality: {
+                only_integer: true,
+                less_than_or_equal_to: 2147483647
```

Comment:
I can convert it into double, but right now I believe that 2147483647 is big enough :smile: